test/hello.jl

```diff
@@ -21,6 +21,7 @@ result = run(sess, a+b)
 #basic tensor facts
 tens = TensorFlow.ones((5,5))
```

Comment:
Isn't this just calling `Base.ones((5,5)` which gives back a Matrix of `Int` ?

Should this be `ones(Tensor, (5,5))` ?
And if so then `size` and `length` won't return `Int`s, they will return `Tensor{Int}`s,
because `size(::Tensor)`  is an overload for `shape(::Tensor)`, not `get_shape(::Tensor)`